packages/shared/src/components/PostOptionsReadingHistoryMenu.tsx

```diff
@@ -0,0 +1,156 @@
+import React, { ReactElement, useContext } from 'react';
+import { Item } from '@dailydotdev/react-contexify';
+import dynamic from 'next/dynamic';
+import { QueryClient, QueryKey, useQueryClient } from 'react-query';
+import { ReadHistoryPost } from '../graphql/posts';
+import ShareIcon from '../../icons/share.svg';
+import BookmarkIcon from '../../icons/outline/bookmark_outlined.svg';
+import BookmarkIconFilled from '../../icons/filled/bookmark_filled.svg';
```

Comment:
I'm not sure we need these changed versions actually, we should keep the icon uniform throughout the app for now.

Reply:
Should the Bookmark icon still be color filled once the article is bookmarked?

Reply:
Yes I think it can be the same approach as we use now for the bookmark icon